docs/graphql/core/migrations/advanced/resetting-migrations.rst

```diff
@@ -0,0 +1,84 @@
+.. meta::
+   :description: Resetting Hasura migrations
+   :keywords: hasura, docs, migration, reset migrations, clear migrations
+
+.. _reset_migration:
+
+Resetting Hasura migrations
+===========================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 1
+  :local:
+
+Introduction
+------------
+
+This page explains how to reset the state of migrations and create new migrations from the state that is on the server. 
+This can be useful if the current migration state on your local machine is corrupted.
+
+Step 1: Delete the local migrations
+-----------------------------------
+
+From your local project directory, delete all the contents in the ``migrations`` directory using the following command:
+
+.. code-block:: bash
+
+   rm -r migrations/*
+
+.. note::
+
+   Before deleting the local migrations, you might consider storing them in a backup folder in case you need them later using the following command:
+
+.. code-block:: bash
+   
+   mv -r migrations migrations_bckup
```

Comment:
It looks a little weird to have this **after** a note. We can either remove the note or put the command in the note also.

Reply:
Alright @marionschleifer